CMakeLists.txt

```diff
@@ -100,7 +100,7 @@ add_library(ViveLoaderLib STATIC
     VRSettings.h)
 target_link_libraries(ViveLoaderLib
     PUBLIC
-    OpenVRDriver osvr::osvrUtil linkable_into_dll
+    OpenVRDriver osvr::osvrUtil linkable_into_dll ${CMAKE_DL_LIBS}
```

Comment:
As Far As I Know, CMake will return a null string on Win32.
And -ldl on OSX/Linux.